src/ui/Controllers/CourseController.cs

```diff
@@ -362,6 +364,12 @@ public async Task<IActionResult> FeesPost(string providerCode, string accreditin
             return RedirectToAction("Show", new { providerCode = providerCode, accreditingProviderCode = accreditingProviderCode, courseCode });
         }
 
+        [Route("{providerCode}/course/{accreditingProviderCode=self}/{courseCode}/vacancies")]
+        public async Task<IActionResult> Vacancies(string providerCode, string accreditingProviderCode, string courseCode)
+        {
+          return frontendUrlService.RedirectToFrontend("/organisations/" + providerCode + "/courses/" + courseCode + "/vacancies");
```

Comment:
suggest
`$"/organisations/{providerCode}/courses/{courseCode}/vacancies"`

Reply:
Sure but this is fine :D